guides/gatherers.md

```diff
@@ -327,14 +344,20 @@ values:
 expectations:
   - name: compare_package_corosync
     expect: facts.compare_package_corosync == values.greater_than_installed
+  
+  - name: compare_package_corosync
+    expect: facts.package_corosync.first().version == "2.4.5"
+
+  - name: sbd_version_same_on_all_hosts
+    expect_same: facts.package_sbd.firts().version
 ```
 
 Example arguments:
 
-| Name                 | Return value                                                 |
-| :------------------- | :----------------------------------------------------------- |
-| `package_name`       | a string containing the installed version of the rpm package |
-| `package_name,2.4.5` | an integer with a value of `-1`, `0` or `-1` (see above)     |
+| Name                 | Return value                                                                  |
+| :------------------- | :---------------------------------------------------------------------------- |
+| `package_name`       | a list containing information about the installed versions of the rpm package |
+| `package_name,2.4.5` | an integer with a value of `-1`, `0` or `-1` (see above)                      |
```

Comment:
typo - the values are '-1', '0' and '**1**'

Reply:
Thanks! Totally missed :sweat_smile: